LearningJourney/Modules/Features/JLibrary/JLibraryTests/Sources/Scenes/ObjectivesLisst/Domain/UseCases/ToggleLearnUseCaseTests.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  ToggleLearnUseCaseTests.swift
+//  JLibraryTests
+//
+//  Created by Vinícius Couto on 05/10/21.
+//
+
+import XCTest
+@testable import JLibrary
+
+final class ToggleLearnUseCaseTests: XCTestCase {
+    private let libraryRepositorySpy = LibraryRepositorySpy()
```

Comment:
Add a mark to improve code organization
```suggestion
    // MARK: - Properties
    
    private let libraryRepositorySpy = LibraryRepositorySpy()
```